ElectronicObserver/Utility/EOMediaPlayer.cs

```diff
@@ -66,12 +66,29 @@ public string SourcePath
 		{
 			if (AudioFile?.FileName != value && !string.IsNullOrEmpty(value))
 			{
-				AudioFile = new(value);
+				AudioFile = new(value)
```

Comment:
Previous AudioFile probably should be disposed before creating a new one ?

Reply:
Fixed